com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariables.hlsl

```diff
@@ -298,10 +298,15 @@ float GetIndirectSpecularMultiplier(uint renderingLayers)
 
 // Functions to clamp UVs to use when RTHandle system is used.
 
-float2 ClampAndScaleUV(float2 UV, float2 texelSize, float numberOfTexels)
+float2 ClampAndScaleUV(float2 UV, float2 texelSize, float numberOfTexels, float2 multiplier)
```

Comment:
A super nitpicky point here, but to be consistent with how we call these things, I'd prefer `scale `here

Reply:
Done